MANIFEST.in

```diff
@@ -1,2 +1 @@
-recursive-exclude niworkflows/ conftest.py
```

Comment:
> I also think it's a mistake to remove conftest.py. Tests are first-class citizens in niworkflows, and should work from an installed copy using pytest --pyargs niworkflows. Removing conftest.py in sdists will modify the behavior of tests depending on the installation method.

I guess then this:
```
[options.packages.find]
exclude =
    *.tests
```
 should also go away? Otherwise, the tests themselves will not be (and currently are not) included in the distribution.

That said, IMHO tests should not be part of the distribution. If they are part of the distribution, it is harder to have a clear picture of what code is actually being tested (a better explanation of this point is given [here](https://hynek.me/articles/testing-packaging/)). And we've already been bitten by that with coverage - it was really painful to set it up. NiWorkflows is far from being able to completely excise the tests out of the distribution tree, and I would probably work towards other goals first, but that is very different from adding the conftest.py modules (and the test modules with them).

Reply:
I agree with @oesteban - I don't see a reason to include them given that our unique dev count is so low. 99.9% of users downloading a distribution will just be using this as the glue for other nipreps

Reply:
Let's not change this for now. I have opened #526 to follow up on this conversation.